### PR TITLE
shooter, intake, transport, and drive all work now

### PR DESCRIPTION
**Pull request by RoyalT-Wrecks**
made minor adjustments that fixed problems.

### CODE REVIEW

File: ChronosTankDrive/src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -84,10 +79,10 @@
   Timer velocityTimer = new Timer();
 
   // DriveTrain Pneumatics
-  Solenoid driveSol = new Solenoid(1, PneumaticsModuleType.CTREPCM, 1);
+  DoubleSolenoid driveSol = new DoubleSolenoid(1, PneumaticsModuleType.CTREPCM, 1, 1);
```
Comment (crschardt) [inline ChronosTankDrive/src/main/java/frc/robot/Robot.java:82]:
```suggestion
  DoubleSolenoid driveSol = new DoubleSolenoid(1, PneumaticsModuleType.CTREPCM, 1, 1); // TODO: fix this call!
```

**Review by IrishConman — APPROVED** [review]

File: ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/Shooter.java [hunk 2]
```diff
@@ -47,7 +48,7 @@ public Shooter(int kShooterPort, int kickerPort, double increment, GenericHID jo
         kFF = Constants.Shooter.sparkmax_kFF;
         kMaxOutput = Constants.Shooter.sparkmax_kMaxOut;
         kMinOutput = Constants.Shooter.sparkmax_kMinOut;
-        maxRPM = 5700;
+        maxRPM = 2300;
```
Comment (crschardt) [inline ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/Shooter.java:51]:
```suggestion
        maxRPM = 5700;  // max free rpm for NEO motor
```

File: ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/ShooterSupersystem.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        shooter.init();
+        hood.init();
+
+    }
+
+    @Override
+    public void periodic() {
+        boolean shooting=false;
+        if (joystick.getRawButton(B_SHOOTER_FAILSAFE)) {
+            hood.setAngle(15);
```
Comment (crschardt) [inline ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/ShooterSupersystem.java:36]:
```suggestion
            // assumes driver has parked robot against dasher board under hub
            hood.setAngle(12);  // launch angle of 78 deg, found by limited testing tonight
```

File: ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/ShooterSupersystem.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            shooter.shoot(2500);
```
Comment (crschardt) [inline ChronosTankDrive/src/main/java/frc/robot/subsystems/shooter/ShooterSupersystem.java:37]:
```suggestion
            shooter.shoot(2300); // found by limited testing tonight
```

**Review by crschardt — CHANGES_REQUESTED** [review]

**Comment by crschardt** [comment]
I identified a few errors that should be fixed in this merge.

**Pull request merged**